javarosa/src/main/java/org/javarosa/xpath/expr/XPathPositionFunc.java

```diff
@@ -54,4 +54,9 @@ private static Double position(TreeReference refAt) {
         return new Double(refAt.getMultLast());
     }
 
+
+    @Override
+    public String getDocumentation() {
+        return "?";
```

Comment:
I don't really know all the intricacies of the `position` function and there is no documentation for it on the CommCare functions confluence page. If anyone else wants to take a crack at documenting it, that would be great